src/webui/webapplication.cpp

```diff
@@ -449,6 +454,8 @@ Http::Response WebApplication::processRequest(const Http::Request &request, cons
     }
     catch (const HTTPError &error) {
         status(error.statusCode(), error.statusText());
+        if (dynamic_cast<const MethodNotAllowedHTTPError*>(&error))
+            header(QLatin1String(Http::HEADER_ALLOW), error.message());
```

Comment:
The HTTP spec requires us to send the `Allow` header when responding with a 405.

Reply:
I'd just check the statusCode instead of a heavy dynamic_cast.